requirements.txt

```diff
@@ -0,0 +1,4 @@
+PyYAML
+git+https://github.com/etingof/pysnmp.git@master#egg=pysnmp
+pyasyncore;python_version>="3.12"
+scapy
```

Comment:
scapy is not used at the end and should be removed; also adding a newline at the end of the file would be good.